app/components/chat/Chat.client.tsx

```diff
@@ -576,9 +591,9 @@ export const ChatImpl = memo(({ description, initialMessages, setInitialMessages
           {
             id: `3-${new Date().getTime()}`,
             role: 'user',
-            content: `[Model: ${model}]\n\n[Provider: ${provider.name}]\n\n[Attachments: ${JSON.stringify(
+            content: `[Model: ${firstChatModel.model}]\n\n[Provider: ${firstChatModel.provider.name}]\n\n[Attachments: ${JSON.stringify(
               attachmentList,
-            )}]\n\n${messageContent}`,
+            )}]\n\n${messageContent}${nextActionSuggestion && `\n\n<think>Proceed with the following task first.\n${nextActionSuggestion}</think>`}`,
```

Comment:
In theory, `<think>` seems to be fit for `assistant` rather than `user`.  is there any reason to treat as `user` message? if not, how about separating that to next message?